### PR TITLE
Replace isparta-instrumenter-loader by isparta-loader for better cove…

### PR DESCRIPTION
**Pull request by zckrs**
…rage support

### CODE REVIEW

**Comment by zckrs** [comment]
See https://github.com/FountainJS/generator-fountain-karma/pull/6

File: generators/app/conf.js [hunk 1]
```diff
@@ -142,7 +142,7 @@ module.exports = function webpackConf(props) {
     conf.module.loaders.push({
       test: lit`/\\.js$/`,
       exclude: lit`/(node_modules|.*\\.spec\\.js)/`,
-      loader: 'isparta-instrumenter'
+      loader: 'isparta-loader'
```
Comment (Swiip) [inline generators/app/conf.js]: If I may, you don't need to add the suffix "loader" `loader: 'isparta'` will do the same

Comment (zckrs) [inline generators/app/conf.js]: Done

**Comment by Swiip** [comment]
Should have merged in the wrong order, you can merge?

**Comment by zckrs** [comment]
My mistake.

Rebased!

**Pull request merged**